snooty/types.py

```diff
@@ -178,7 +178,7 @@ class TargetDatabase:
     """A database of targets known to this project."""
 
     intersphinx_inventories: Dict[str, intersphinx.Inventory]
-    local_definitions: Dict[str, str]
+    local_definitions: Dict[str, Tuple[FileId, str]]
```

Comment:
What is the purpose of the `str` in the tuple? It looks like it is only ever set to an empty string, and never accessed.

Reply:
ugh forgot to fill that in. The str is meant to point to the ID of the element to link to

Reply:
@sophstad Does the frontend have enough information to generate a complete link based on the page name and target?

Reply:
@i80and Based on our discussion last week, I thought more would be implemented in the parser layer. Namely, I thought that the parser would be responsible for fetching the ID's title (the link text) and including it in the node. I do think that is a little bit much for the front end to take on, but am open to other approaches!

Reply:
Fetching the title will be done in the formatting ticket, I think! My concern is about the href right now

Reply:
Oh gotcha! Yes, this looks adequate for the frontend.